packages/react/src/date-picker/reactDayPickerStyles.ts

```diff
@@ -83,6 +83,8 @@ export const reactDayPickerStyles = (range: boolean) =>
 			},
 			'&:not([disabled]):hover': {
 				backgroundColor: boxPalette.backgroundShade,
+				color: boxPalette.foregroundText,
+				textDecoration: 'underline',
```

Comment:
```suggestion
				...packs.underline,
```

Could we use this instead please? I know it's a bit inconsistent at the moment, but would be good to slowly fix this as we go.

Reply:
But we can also do this another time, not really a big deal.